### ISSUE TITLE
why the loss not converge?

### ISSUE THREAD

**Issue by daiwc** [jwchoi384/Gaussian_YOLOv3#57]
I used the Gaussian_yolov3_KITTI_3cls to train a new model without using pretrained model, the loss still around a hundred after 10000 iterations.

**Comment by jwchoi384** [comment]
@daiwc 
Hello, please see the link below.
I have already explained this issue elsewhere.
https://github.com/jwchoi384/Gaussian_YOLOv3/issues/19#issuecomment-553878638
https://github.com/jwchoi384/Gaussian_YOLOv3/issues/54#issuecomment-573081226

**Comment by daiwc** [comment]
thanks a lot!



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Jiwoong Choi"<notifications@github.com&gt;; 
发送时间: 2020年2月12日(星期三) 下午4:32
收件人: "jwchoi384/Gaussian_YOLOv3"<Gaussian_YOLOv3@noreply.github.com&gt;; 
抄送: "戴伟聪"<493980892@qq.com&gt;; "Mention"<mention@noreply.github.com&gt;; 
主题: Re: [jwchoi384/Gaussian_YOLOv3] why the loss not converge? (#57)




@daiwc
 Hello, please see the link below.
 I have already explained this issue elsewhere.
 #19 (comment)
 #54 (comment)
